rotors_description/urdf/iris.xacro

```diff
@@ -75,8 +73,6 @@
     motor_number="0"
     rotor_drag_coefficient="${rotor_drag_coefficient}"
     rolling_moment_coefficient="${rolling_moment_coefficient}"
-    mesh="iris_prop"
-    mesh_scale="${mesh_scale_prop}"
```

Comment:
if you want your own props, which I would appreciate, then we should add this parameter to the `vertical_rotor` macro. I guess you had it in there at some point :)